src/stack.jl

```diff
@@ -31,29 +24,69 @@ end
 Stack{T}() where {T} = Stack(Deque{T}())
 Stack{T}(blksize::Integer) where {T} = Stack(Deque{T}(blksize))
 
+"""
+    isempty(s::Stack)
+
+Check if stack `s` is empty.
+"""
 Base.isempty(s::Stack) = isempty(s.store)
+
+
+"""
+    isempty(s::Stack)
+
+Return the number of elements in stack `s`.
+"""
 Base.length(s::Stack) = length(s.store)
-Base.eltype(::Type{Stack{T}}) where T = T
+
+
+"""
+    eltype(::Type{Stack{T}}) where {T}
+
+Return the type of the elements in the stack.
+"""
```

Comment:
```suggestion
```

Per method dostrings are not needed when they add no value over the per function ones.
Plus this one is wrong